api/v1beta1/tags.go

```diff
@@ -95,11 +99,37 @@ func (t Tags) Validate() []*field.Error {
 				field.Invalid(field.NewPath("spec", "additionalTags"), v, "value cannot be longer than 256 characters"),
 			)
 		}
+		if isUserOwnedTag(k) {
```

Comment:
Before here, we should return error if tag starts with `aws:` and here just count all the tags.